pvlib/tests/test_modelchain.py

```diff
@@ -447,22 +447,29 @@ def acdc(mc):
     mc.ac = mc.dc
 
 
+# TODO in v0.9: remove 'snlinverter', 'adrinverter', 'pvwatts'
```

Comment:
Same here about 'pvwatts' being removed

Reply:
I've edited the TODO comment to drop the 'snlinverter' and 'adrinverter' values for `ModelChain.ac_model`. 

There's a loose thread outside of this PR, in that `PVSystem.snlinverter`, `PVSystem.adrinverter` and `PVSystem.pvwatts_ac` methods aren't renamed. I couldn't come up with a naming scheme for the `PVSystem` methods that I liked, when the methods wrap functions in a different module. The method name needs to package both the method purpose (inverter model) and the model (snl, adr, pvwatts).